examples/grpc_proxying/helloworld_service_pb2_grpc.py

```diff
@@ -0,0 +1,70 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+"""Client and server classes corresponding to protobuf-defined services."""
+import grpc
+
+import helloworld_service_pb2 as helloworld__service__pb2
+
+
+class HelloWorldServiceStub(object):
+    """The greeting service definition.
+    """
+
+    def __init__(self, channel):
+        """Constructor.
+
+        Args:
+            channel: A grpc.Channel.
+        """
+        self.SayHello = channel.unary_unary(
+                '/HelloWorldService/SayHello',
+                request_serializer=helloworld__service__pb2.HelloRequest.SerializeToString,
+                response_deserializer=helloworld__service__pb2.HelloReply.FromString,
+                )
+
+
+class HelloWorldServiceServicer(object):
+    """The greeting service definition.
+    """
+
+    def SayHello(self, request, context):
+        """Sends a greeting
+        """
+        context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+        context.set_details('Method not implemented!')
+        raise NotImplementedError('Method not implemented!')
+
+
+def add_HelloWorldServiceServicer_to_server(servicer, server):
+    rpc_method_handlers = {
+            'SayHello': grpc.unary_unary_rpc_method_handler(
+                    servicer.SayHello,
+                    request_deserializer=helloworld__service__pb2.HelloRequest.FromString,
+                    response_serializer=helloworld__service__pb2.HelloReply.SerializeToString,
+            ),
+    }
+    generic_handler = grpc.method_handlers_generic_handler(
+            'HelloWorldService', rpc_method_handlers)
+    server.add_generic_rpc_handlers((generic_handler,))
+
+
+ # This class is part of an EXPERIMENTAL API.
+class HelloWorldService(object):
+    """The greeting service definition.
+    """
+
+    @staticmethod
+    def SayHello(request,
+            target,
+            options=(),
+            channel_credentials=None,
+            call_credentials=None,
+            insecure=False,
+            compression=None,
+            wait_for_ready=None,
+            timeout=None,
+            metadata=None):
+        return grpc.experimental.unary_unary(request, target, '/HelloWorldService/SayHello',
```

Comment:
I'm a bit concerned about this breaking our automated tests -- specifically the `validate examples` workflow we run via Mechnical Markdown.

We will certainly be updating the GRPC library as new versions come out. I assume given the experimental nature of this feature the maintainers of the GRPC library may be moving / removing this feature in the future.

Is there a different way to accomplish what you need to accomplish without the use of an experimental API?

Reply:
This is what is currently being generated using gRPC tools. I don't have control over this part. I have to mention this is part of the example and not part of the gRPC extension.

Reply:
Yeah, this is something internal to the generated code. The main dapr API's grpc generated code has this too.